packages/appium/lib/appium.js

```diff
@@ -195,6 +196,16 @@ class AppiumDriver extends DriverCore {
     }));
   }
 
+  async getAppiumSessions () {
+    throw new errors.NotImplementedError('Not implemented yet. ' +
+      'Please check https://github.com/appium/appium/issues/20880 for more details.');
+  }
+
+  async getAppiumSessionCapabilities () {
+    throw new errors.NotImplementedError('Not implemented yet. ' +
+      'Please add getAppiumSessionCapabilities method in the unbrella driver.');
```

Comment:
it's already there %)

Reply:
oh, nice

Reply:
oh, nice

Reply:
I meant the error message itself is confusing , since this method is already defined in the umbrella driver. It makes sense to keep the definition though

Reply:
I see.. this method needs a way to call the same name's one from destination driver like delete session since not this method is part of https://github.com/appium/appium/pull/20936/files#diff-ad0e9402cbbaa56d0519b416feaff4f66815b49e7cef7518c049c12fc0c1cadcR966 ... I didn't catch up with the method call route.
So, this method is not well implemented yet. Probably here needs similar get destination driver instance one like delete session does, whcih probably nice to extract similar method and re-use for both.


I'd like to do this implementation in a followup PR with `endpointRequiresSessionId`'s change (to not grow this pr diff)